Core/DetectorConstruction.cpp

```diff
@@ -879,6 +937,19 @@ void DetectorConstruction::ConstructTargetRun7()
     transform, xadFilling_logical, "xadFillingGeom",
     fWorldLogical, true, 0, checkOverlaps
   );
+  
+  // Vacuum
+  G4Tubs* smallChamberRun7_vac = new G4Tubs(
+    "smallChamberRun7_vac", 0 * cm, 0.53 * cm, 2.7 * cm , 
```

Comment:
Put those 0.53*cm and 2.7*cm value as variables as in functions for constructing other targets.
You can also comment why these value are as it is (probably 0.54 - 0.01 and 2.8 - 0.1, right?)

Reply:
Can I put the variables for the value of the inner and outer radius of the target chamber in run7? As the vacuum values are also related to those of the chamber dimension.

Reply:
Yes, sure